db/seeds.rb

```diff
@@ -6,11 +6,15 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-Video.create(title: "The Wire", description: "One of the best TV shows ever made.", large_cover_url: "wire.jpg", category_id: 3)
-Video.create(title: "Sopranos", description: "A very entertaining and well made TV show.", large_cover_url: "sopranos.jpg", category_id: 1)
-Video.create(title: "The Simpsons", description: "A classic that needs no description.", large_cover_url: "simpsons", category_id: 2)
+dramas = Category.create(name: "Drama")
+cartoon = Category.create(name: "Cartoon")
+detective = Category.create(name: "Detectice")
 
-Video.create(title: "Monk", description: "A neurotic detective.", large_cover_url: "monk.jpg", small_cover_url: "monk_small.jpg", category_id: 3)
+Video.create(title: "The Wire", description: "One of the best TV shows ever made.", large_cover_url: "wire.jpg", small_cover_url: "wire_small.jpg", category_id: 3)
```

Comment:
Always use `create!` instead of `create` in seeds. The bang version will notify you when the record you try to create is not valid. Will help you a lot when something goes wrong and you'll have to debug.